src/components/spatial/Spatial.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 
-import { ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM } from 'deck.gl';
-import { SelectablePolygonLayer } from '../../layers';
+import {
+  ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM, BitmapLayer,
+} from 'deck.gl';
+import { SelectablePolygonLayer, IdentityCoordinatesTileLayer } from '../../layers';
 import { cellLayerDefaultProps, PALETTE, DEFAULT_COLOR } from '../utils';
 import AbstractSelectableComponent from '../AbstractSelectableComponent';
 import LayersMenu from './LayersMenu';
-import OpenSeadragonComponent from '../../vendor/OpenSeadragonComponent';
 
 export function square(x, y, r) {
   return [[x, y + r], [x + r, y], [x, y - r], [x - r, y]];
 }
 
+function loadImage(src) {
+  // This function replaces load from loaders.gl (7.3.5) which was not working
+  // with this version of deckgl (7.1.4).
```

Comment:
Link to any relevant issues.

Reply:
There are none - it was just easier to replace and move on, especially with our very custom set-up right now